lightning/src/events/mod.rs

```diff
@@ -884,6 +884,7 @@ impl Writeable for Event {
 					(1, None::<NetworkUpdate>, option), // network_update in LDK versions prior to 0.0.114
 					(2, payment_failed_permanently, required),
 					(3, false, required), // all_paths_failed in LDK versions prior to 0.0.114
+					(4, path.blinded_tail, option),
```

Comment:
The intention is to only break backwards compat once a user proactively opts into/uses BOLT 12. This will only write the even tlv if `path.blinded_tail` is `Some`.